debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -46,6 +47,7 @@
 public class PostgresChangeRecordEmitter extends RelationalChangeRecordEmitter {
 
     private final ReplicationMessage message;
+    private final Partition partition;
```

Comment:
I think I saw a pattern somewhere in the codebase where multiple classes in the hierarchy had their own private property referencing the same object instead of using a protected property in the base class, so I did the same. Now the property in the base class is protected.